core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -494,6 +495,19 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
           }
         }
         .flatMap { entry => EventLogFileReader(fs, entry) }
+        .filter { reader =>
+          try {
+            reader.modificationTime
+            true
+          } catch {
+            case e: Exception =>
+              logError("Exception in getting modificationTime of " + reader.rootPath.getName + ".")
```

Comment:
`logError` is too alarming to the users. I guess `logInfo` would be enough because this will be a new message to the users.

Reply:
changed to logInfo as sugested.